cmd/guacone/cmd/oci.go

```diff
@@ -131,6 +142,90 @@ var ociCmd = &cobra.Command{
 	},
 }
 
+var ociRegistryCmd = &cobra.Command{
+	Use:   "registry [flags] registry",
+	Short: "takes an OCI registry with catalog capability and downloads sbom and attestation stored in OCI to add to GUAC graph",
+	Args:  cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		opts, err := validateOCIRegistryFlags(
+			viper.GetString("gql-addr"),
+			viper.GetString("header-file"),
+			viper.GetString("csub-addr"),
+			viper.GetBool("csub-tls"),
+			viper.GetBool("csub-tls-skip-verify"),
+			viper.GetBool("add-vuln-on-ingest"),
+			viper.GetBool("add-license-on-ingest"),
+			viper.GetBool("add-eol-on-ingest"),
```

Comment:
linking https://github.com/guacsec/guac/pull/2265 to track concurrent changes for the additional add-depsdev-on-ingest flag

Reply:
thanks for flagging this, that PR has since merged so I've fixed the conflicts on this branch here.